torchvision/prototype/transforms/functional/_utils.py

```diff
@@ -1,29 +0,0 @@
-from typing import Tuple, Union, cast
-
-import PIL.Image
-import torch
-from torchvision.prototype import features
-from torchvision.transforms import functional_tensor as _FT, functional_pil as _FP
-
-
-def get_image_size(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> Tuple[int, int]:
```

Comment:
These methods look like a remnant of "high-level" kernels. Even if we have maintained them, I believe we need to move them out of `functional` because they look more like utilities than kernels.